opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/jms/JmsArtemisManualServerTest.java

```diff
@@ -73,10 +72,8 @@ public void start() throws Exception {
         org.apache.activemq.artemis.core.config.Configuration configuration = new ConfigurationImpl();
 
         HashSet<TransportConfiguration> transports = new HashSet<>();
-        transports.add(new TransportConfiguration(NettyAcceptorFactory.class.getName()));
```

Comment:
@alesj why it was here? When the test was executed a couple of times in sequence it failed on some unrelated errors therefore I have removed it.

Reply:
This was a copy/paste from Artemis docs/examples. Didn't bother further. :-)